### DISCUSSION TITLE
NRF24L01 Module > litte mistake in docs and... not sure how to use the NRF24L01 module

### DISCUSSION THREAD

**Discussion by Roellen** [arendst/Tasmota#16066] [Q&A] [answered]
### PROBLEM DESCRIPTION
1. Little mistake in the documents
Choosing "SPI DC/SPI CS" will never work. 
To make this NRF24L01 module work... "NRF24 DC/NRF24 CS" needs to be selected. Maybe it helps other people to update the docs here. 

2. I can't find any info on how to use the NRF24L01 Module? I thought I can "read" and "send" commands with the NRF24L01 module... just like the Sonoff RF Bridge. But I can't find any info on how to use the module? I thought that maybe a small piece of code will be send over MQTT. And if I would send the command back... then the NRF24L01 module will send this command..?**

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x ] Device used (e.g., Sonoff Basic): _____
- [ x] Tasmota binary firmware version number used: 8.1.0.2
  - [ ] Pre-compiled
  - [x ] Self-compiled
- [ ] Flashing tools used:online
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by barbudor** [comment, marked as answer]
1. You can submit doc changes by clicking on the pencil button on the top left. Docs are hosted in a different repository (https://github.com/tasmota/docs)

2. As per the [NRF24L01 doc page](https://tasmota.github.io/docs/NRF24L01/) Tasmota is using the NRF only for receiving some BLE sensors, not for general bi directional communication


Also please do not open an issue for asking for help. Discussions are meant for that.
Issues are for qualified bugs.
